factory/ideas.php

```diff
@@ -155,6 +159,20 @@ public function get_ideas($number = 10, $sort = 'date', $sort_direction = 'DESC'
 		// Prepend $status to our $where clause
 		$where = $status . (($where) ? ' AND ' . $where : '');
```

Comment:
Can we type hint it as an int here for redundancy? It will convert to string straight afterwards anyway when the string concat is done.